diff --git a/node_modules/concat-md/dist/index.js b/node_modules/concat-md/dist/index.js
index 8ec2c37..de01f3f 100644
--- a/node_modules/concat-md/dist/index.js
+++ b/node_modules/concat-md/dist/index.js
@@ -44,12 +44,49 @@ exports.gitHubLink = gitHubLink;
 function arrify(input) {
     return Array.isArray(input) ? input : [input];
 }
+
+/**
+ *  ...images
+ */
+
+function transformImages(input, transform) {
+   const linkRegExp = /\!\[([^\]]+)?\]\(([^)]+)\)/g; //![image name](imageurl)
+   let output = input;
+   let match;
+ 
+   // Transforming basic links
+   while ((match = linkRegExp.exec(input)) !== null) {
+     const string = match[0];
+     const text = match[1] || '';
+     const url = match[2];
+ 
+     /**
+      * Callback to transform a link within a Markdown string.
+      * @callback TransformCallback
+      * @param {String} link - URL of the link.
+      * @param {String} text - Text of the link.
+      * @returns {?String} Modified URL.
+      */
+     const replacement = transform(url, text);
+ 
+     if (replacement) {
+       output = output.replace(string, `![${text}](${replacement})`);
+     }
+   }
+
+   return output;
+ };
+ 
+
+
+
 /**  @ignore */
 class MarkDownConcatenator {
     constructor(dir, { toc = false, tocLevel = 3, ignore = [], decreaseTitleLevels = false, startTitleLevelAt = 1, joinString = "\n", titleKey, dirNameAsTitle = false, fileNameAsTitle = false, } = {}) {
         this.visitedDirs = new Set();
         this.fileTitleIndex = new Map();
         this.files = [];
+        this.orgDir = dir.replace(/.*\//g, "");
         this.dir = dir;
         this.toc = toc;
         this.tocLevel = tocLevel;
@@ -81,7 +118,7 @@ class MarkDownConcatenator {
         return fileNames.map((fileName) => ({ path: fileName, ...front_matter_1.default(fs_1.readFileSync(fileName, { encoding: "utf8" })) }));
     }
     getDirParts(file) {
-        return this.dir === path_1.dirname(file.path) ? [] : path_1.relative(this.dir, path_1.dirname(file.path)).split(path_1.sep);
+        return this.dir === path_1.dirname(file.path) ? [] : path_1.relative(this.dir, path_1.dirname(file.path)).replace(/^(.?)_/,'').split(path_1.sep);
     }
     addTitle(file) {
         let titleMd = "";
@@ -140,6 +177,34 @@ class MarkDownConcatenator {
         }
         return result;
     }
+    modifyImages(file){
+      
+        return transformImages(file.body, (image) => {
+            if (image.startsWith("http")) {
+                return image;
+            }
+        
+            const absoluteTargetPath = path_1.join(path_1.dirname(file.path), image);
+            const relativeTargetPath = path_1.relative(this.dir, absoluteTargetPath);
+            const hashPosition = absoluteTargetPath.indexOf("#");
+            const hash = hashPosition > -1 ? absoluteTargetPath.slice(hashPosition) : "";
+            const targetFile = hashPosition > -1 ? absoluteTargetPath.slice(0, hashPosition) : absoluteTargetPath;
+            
+            console.log(process.env)
+           
+            try {
+                const newImage = `https://github.com/${process.env.npm_package_name}/${process.env.LOGNAME}/blob/main/` + this.orgDir + "/" + relativeTargetPath +"?raw=true";
+                
+                return newImage;
+            }
+            catch (e) {
+                /* istanbul ignore next */
+                return "";
+            }
+            //console.log(link, newLink, targetFile, hash);
+        });
+    }
+
     modifyLinks(file) {
         return transformLinks(file.body, (link) => {
             if (link.startsWith("http")) {
@@ -175,6 +240,11 @@ class MarkDownConcatenator {
         files.forEach((file) => {
             file.body = this.modifyLinks(file); // eslint-disable-line no-param-reassign
         });
+        // 3rd pass loop is necessary, because all titles has to be processed.
+        files.forEach((file) => {
+            file.body = this.modifyImages(file); // eslint-disable-line no-param-reassign
+        });
+
         const result = files.map((file) => file.body).join(this.joinString);
         return this.addToc(result);
     }
